frontend/web/components/modals/CreateSegment.js

```diff
@@ -435,10 +435,12 @@ const LoadingCreateSegment  = (props) => {
     const [segmentData, setSegmentData] = useState(null);
 
     useEffect(()=>{
-        _data.get(`${Project.api}projects/${props.projectId}/segments/${props.segment}`).then((segment)=> {
-            setSegmentData(segment);
-            setLoading(false)
-        })
+        if(props.segment) {
```

Comment:
Didn't we add this recently? Do you need to merge main back into this branch perhaps?